contribute/project-ideas.md

```diff
@@ -206,3 +126,64 @@ problems. It would be great to have a better solution here. Ideally osm2pgsql
 could just consume the diffs downloaded from planet.osm.org and do the right
 thing.
 
+[Discussion](https://github.com/osm2pgsql-dev/osm2pgsql/issues/1248){:.extlink}
+
+## Resuming imports/updates
+
+Imports with osm2pgsql can take quite some time and if something breaks you
+have to start from scratch. Some kind of "resume" feature is requested often
+and we should look into what's needed for that.
+
+We'd have to define checkpoints and make sure that all data is written to disk
+at that point. Then on restart osm2pgsql has to detect that it failed and
+resume at the correct point. In
+[some situations](https://github.com/osm2pgsql-dev/osm2pgsql/issues/799){:.extlink}, the
+user might also want to start osm2pgsql processing at a well defined point.
+
+[Discussion](https://github.com/osm2pgsql-dev/osm2pgsql/issues/1755){:.extlink}
+
+## Processing Flexibility
+
+Osm2pgsql processes data in several steps: Database tables are created, then
+data is imported, then clustered, then indexes created. Some of those steps are
+already configurable, but there is a lot of hardcoded logic in here. Users have
+repeatedly asked for more flexibility, for instance for index creation, which
```

Comment:
Might need some rephrasing. It says users ask for flexibility but it's also much more flexible already.

Reply:
Good catch. That one was meant to be removed. Done now.